### PR TITLE
Multiple visu same page

### PR DESCRIPTION
**Pull request by mmilaprat**
This PR is related with issue policycompass/issues/672

Is to enable to link n times the same visualisation in stories

### CODE REVIEW

File: app/modules/visualization/directives/visualizationDirective.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ angular.module('pcApp.visualization.directives.visualizationDirective', [
                     $scope.idvisulist = $scope.visualizationId;
                 }
                 $routeParams.visualizationId = $scope.visualizationId;
+                $routeParams.randomID = Math.floor(Math.random()*16777215);
```
Comment (cbotsikas) [inline app/modules/visualization/directives/visualizationDirective.js]: Why don't you use $scope here?

File: app/modules/visualization/controllers/visualizationControllers.js [hunk 2]
```diff
@@ -3613,7 +3613,14 @@ angular.module('pcApp.visualization.controllers.visualization', [
                 $routeParams.visualizationId = $scope.idvisulist;
                 $scope.list = true;
             }
-
+			
+			if ($routeParams.randomID) {
+				$scope.randomID = $routeParams.randomID;
+			}
+			else {
+				$scope.randomId = $routeParams.visualizationId;
```
Comment (cbotsikas) [inline app/modules/visualization/controllers/visualizationControllers.js]: randomId  != randomID

File: app/modules/visualization/js/objectPie.js [hunk 3]
```diff
@@ -145,6 +145,7 @@ policycompass.viz.pie = function (options) {
         })
 		.on("mouseout", function (d, i) {
                 self.clicToOpen = true;
+                tooltip.style("opacity", 1.0).html("");
```
Comment (cbotsikas) [inline app/modules/visualization/js/objectPie.js]: why do you change opacity to 1 and then on next line to 0?

File: app/modules/visualization/partials/graph.html [hunk 4]
```diff
@@ -54,18 +54,30 @@
 
     <center>
         <div ng-show="isSelectedSon('graph_pie')">
+        	<!--
             <div class="pcPieChart" dataset="dataset" chartid="visualization.id" show-Legend="showLegend" small="list" mode="mode" show-Labels="showLabels"></div>
+           	-->
```
Comment (cbotsikas) [inline app/modules/visualization/partials/graph.html]: You shouldn't keep your old code in comments. If you need to, you can always check git history.

**Comment by mmilaprat** [comment]
latest commit solve requests

**Comment by mmilaprat** [comment]
@cbotsikas, have you had time to review the last commit?

**Comment by cbotsikas** [comment]
https://github.com/policycompass/policycompass/issues/672

**Pull request merged**